m3u8/model.py

```diff
@@ -624,6 +630,47 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
+class InitializationSection(BasePathMixin):
+    '''
+    Used to obtain Media Initialization Section required to
+    parse the applicable Media Segments (EXT-X-MAP)
+
+    `uri`
+      is a string. ex:: "https://priv.example.com/key.php?r=52"
+
+    `byterange`
+      value of BYTERANGE attribute
+
+    `base_uri`
+      uri the segment comes from in URI hierarchy. ex.: http://example.com/path/to
+
+    '''
+
+    tag = ext_x_map
+
+    def __init__(self, base_uri, uri, byterange=None):
+        self.base_uri = base_uri
+        self.uri = uri
+        self.byterange = byterange
+
+    def __str__(self):
+        output = []
+        if self.uri:
+            output.append('URI=' + quoted(self.uri))
+        if self.byterange:
+            output.append('BYTERANGE=' + self.byterange)
+        return self.tag + ':' + ','.join(output)
```

Comment:
I think you can take advantage of `f strings` here.
```python
return f"{self.tag}:{','.join(output)}"
```

Or even format:

```python
return "{tag}:{attributes}".format(tag=self.tag, attributes=",".join(output))
```

Reply:
String interpolation introduced in python3.6, so I think format form is preferred for legacy interpreter's hostages, like me.

Reply:
You are right!